migrations/023.do.add-per-client-rsr-table.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE daily_client_retrieval_stats (
+  client_id TEXT NOT NULL,
+  day DATE NOT NULL,
```

Comment:
Let's be consistent with the order of columns in the primary key and in the definitions of other "daily" tables.

```suggestion
  day DATE NOT NULL,
  client_id TEXT NOT NULL,
```